custom_typings/vaadin.d.ts

```diff
@@ -0,0 +1,14 @@
+interface Vaadin {
+  developmentModeCallback: {
+    'usage-statistics': () => void;
+    'vaadin-license-checker'?: () => void;
+  };
+  registrations: Array<{ is: string }>;
```

Comment:
These should be optional. Currently getting this:
<img width="889" alt="Screenshot 2020-04-16 11 54 32" src="https://user-images.githubusercontent.com/1222264/79436215-40cc8f00-7fd9-11ea-9944-ffd2feab57a2.png">

Reply:
Done.